app/graph/mutations/update_submission_mutation.rb

```diff
@@ -1,42 +1,45 @@
 # frozen_string_literal: true
 
 module Mutations
-  module UpdateConsignmentSubmission
+  module UpdateSubmissionMutation
     Definition =
       GraphQL::Relay::Mutation.define do
-        name 'UpdateConsignmentSubmission'
+        name 'UpdateSubmissionMutation'
 
         input_field :id, !types.ID
-        input_field :additional_info, types.String
-        input_field :artist_id, types.String
-        input_field :authenticity_certificate, types.Boolean
+        input_field :additionalInfo, types.String
```

Comment:
`graphql-ruby` has a nice (or rather, confusing) feature where it will automatically take `snake_case` fields and camelize them. However, this doesn't apply to input types.